ejb-txn-remote-call/README.adoc

```diff
@@ -0,0 +1,942 @@
+include::../shared-doc/attributes.adoc[]
+
+= ejb-txn-remote-call: Demonstrates remote EJB calls and transaction propagation
+:author: Ondra Chaloupka
+:level: Intermediate
+:technologies: EJB, JTA, Clustering
+
+[abstract]
+The `ejb-txn-remote-call` quickstart demonstrates the remote EJB calls over two application servers of {productName}.
+
+:standalone-server-type: ha
+:archiveType: war
+:uses-h2:
+:requires-multiple-servers:
+:jbds-not-supported:
+
+== What is it?
+
+The `ejb-txn-remote-call` quickstart demonstrates the remote EJB calls over two application servers of {productNameFull}.
+
+== Description
+
+The EJB remote calls contains a JTA transaction which is propagated over the EJB call.
+Further the quickstart demonstrates the transaction recovery which is run for both servers
```

Comment:
I recommend a comma after "further".

"demonstrates the transaction recovery which is"

Need a comma after "recovery".

Reply:
+1